docs/pages/router/reference/not-found.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: Not Found Routes
```

Comment:
```suggestion
title: Not found routes
hideTOC: true
```
Fixed the title casing and added `hideTOC` since this page doesn't have any sections.